docs/REST API/rest-api.md

```diff
@@ -4612,6 +4612,7 @@ Sends a command to lock the specified macOS, iOS, iPadOS, Linux, or Windows host
 
 To lock a macOS, iOS, or iPadOS host, the host must have MDM turned on. To lock a Windows or Linux host, the host must have [scripts enabled](https://fleetdm.com/docs/using-fleet/scripts).
 
+> For iOS and iPadOS, this enables Lost Mode.
```

Comment:
Let's combine with the paragraph above since it's thematically similar (otherwise this will display as a separate callout box) 

```suggestion
To lock a macOS, iOS, or iPadOS host, the host must have MDM turned on. To lock a Windows or Linux host, the host must have [scripts enabled](https://fleetdm.com/docs/using-fleet/scripts). For iOS and iPadOS, this enables Lost Mode.
```